internal/controller/postgresql/postgresqldatabase_controller_test.go

```diff
@@ -263,9 +282,28 @@ var _ = Describe("PostgresqlDatabase tests", func() {
 		Expect(exists).To(BeTrue())
 
 		// Check if roles exists and are granted to default user
-		checkRoleInSQLDb(postgresUser, ownerRole)
-		checkRoleInSQLDb(postgresUser, readerRole)
-		checkRoleInSQLDb(postgresUser, writerRole)
+		checkRoleInSQLDb(ownerRole)
+		checkRoleInSQLDb(readerRole)
+		checkRoleInSQLDb(writerRole)
+
+		// Check role members and rights
+		ownerMemberWithAdminOption, err := getSQLRoleMembershipWithAdminOption(item.Status.Roles.Owner)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(ownerMemberWithAdminOption).To(Equal(map[string]bool{postgresUser: false}))
+		writerMemberWithAdminOption, err := getSQLRoleMembershipWithAdminOption(item.Status.Roles.Writer)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(writerMemberWithAdminOption).To(Equal(map[string]bool{postgresUser: false}))
+		readerMemberWithAdminOption, err := getSQLRoleMembershipWithAdminOption(item.Status.Roles.Reader)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(readerMemberWithAdminOption).To(Equal(map[string]bool{postgresUser: false}))
+
+		Expect(len(item.Status.Schemas)).To(Equal(1))
+		Expect(item.Status.Schemas).To(ContainElement(pgPublicSchemaName))
+
+		// Check schema exists in sql db
+		exists, err = isSQLSchemaExists(pgPublicSchemaName)
```

Comment:
may need a test where schema is set to something different than "public"

Reply:
As discussed, it is already present ;)